www/js/app.js

```diff
@@ -386,6 +397,36 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         }, 600);
     });
 
+    //focus search bar (#prefix) if Home key is pressed
+    function focusPrefixOnHomeKey(event) {
+        //check if home key is pressed
+        if (event.key === 'Home') {
+            //scroll to top of #search-article section if scroll bar is associated with it
+            $('#search-article').scrollTop(0);
+            //in case the scroll bar is associated with iframe #articleContent
+            var iframeArticleDocument = document.getElementById('articleContent').contentWindow.document;
+            $(iframeArticleDocument).scrollTop(0);
```

Comment:
@Hrishi-Raj  You're using a hybrid of native DOM methods and jQuery here. Since you select iframeArticleDocument using the native DOM method, you should also use the native DOM `iframeArticleDocument.scrollTo(0, 0);`

In my opinion this whole function doesn't need jQuery at all.

Reply:
Looking at this more closely, @Hrishi-Raj , I don't think you should add an event listener **every time** the Home key is pressed. This is wasteful. Why can't you just define the required listeners once in the `switchHomeKeyToFocusSearchBar()` function when the user activates this feature?